File: src/main/java/com/terrencewei/markdown/bean/OSSInput.java
package com.terrencewei.markdown.bean;

/**
 * Created by terrencewei on 2017/04/21.
 */
public class OSSInput {

    private String      bucketName;
    private OSSObject[] objects;



    public String getBucketName() {
        return bucketName;
    }



    public void setBucketName(String pBucketName) {
        bucketName = pBucketName;
    }



    public OSSObject[] getObjects() {
        return objects;
    }



    public void setObjects(OSSObject[] pObjects) {
        objects = pObjects;
    }



    @Override
    public String toString() {
        return "OSSInput [" + "bucketName=" + bucketName + ", objects=" + objects + ']';
    }
}

File: VirtualHostsRoot/Aliyun/myfolder/php-my/src/terrencewei/Model/OSSInput.php
<?php

namespace terrencewei\Model;

class OSSInput implements \JsonSerializable
{
    private $bucketName;
    private $objects = array();

    public function jsonSerialize()
    {
        return get_object_vars($this);
    }

    /**
     * @return mixed
     */
    public function getBucketName()
    {
        return $this->bucketName;
    }

    /**
     * @param mixed $bucketName
     */
    public function setBucketName($bucketName)
    {
        $this->bucketName = $bucketName;
    }

    /**
     * @return array
     */
    public function getObjects()
    {
        return $this->objects;
    }

    /**
     * @param array $objects
     */
    public function setObjects($objects)
    {
        $this->objects = $objects;
    }


}